ledger-tool/src/main.rs

```diff
@@ -818,8 +854,35 @@ fn open_blockstore(
         },
     ) {
         Ok(blockstore) => blockstore,
+        Err(BlockstoreError::RocksDb(err))
+            if (err
+                .to_string()
+                .starts_with("Invalid argument: Column family not found:")
+                || err
+                    .to_string()
+                    .starts_with("IO error: No such file or directory:"))
```

Comment:
Haven't touched this part for a while.  Is it the error message for rocksdb not found in the specified path?

Reply:
Yes it is